index.js

```diff
@@ -6,7 +6,7 @@ const util = require('util');
 
 const MSG_INVALID_ALGORITHM = '"%s" is not a valid algorithm.\n  Supported algorithms are:\n  "HS256", "HS384", "HS512", "RS256", "RS384", "RS512" and "none".'
 const MSG_INVALID_SECRET = 'secret must be a string or buffer';
-const MSG_INVALID_KEY = 'key must be a string or buffer';
+const MSG_INVALID_KEY = 'key must be a string, a buffer or an object';
```

Comment:
This message is being reused for both `sign` and `verify`, and as such is not incorrect for `verify`